<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleShips</string>
  </property>
  <widget class="QLabel" name="Field1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Field2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonEnd">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Spiel beenden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonNext">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>NÃ¤chste Zug</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="field1" rowstretch="0" columnstretch="0" rowminimumheight="0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="field2">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="whichTurn"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>189</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="statusLog"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="whichPhase"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>40</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="randPlaceBtn1"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="randPlaceBtn2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
